catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoEntryQueryException.java

```diff
@@ -17,7 +17,7 @@
 /**
  * Thrown by a {@link GeoEntryQueryable} when an error occurs while querying a GeoNames resource.
  */
-public class GeoEntryQueryException extends RuntimeException {
+public class GeoEntryQueryException extends Exception {
```

Comment:
We had a discussion about checked vs unchecked exceptions in this package in a previous PR, and decided that they should be checked.  This exception fell through the cracks and was written to be unchecked, but the rest are checked.